components/footer.js

```diff
@@ -49,7 +48,18 @@ export default function Footer() {
           <p>Einnahmen</p>
         </NavElement>
       </Link>
-    
+      <Link href="/profile">
+        <NavElement active={pathname.includes("/profile")}>
+          <Image
+            src={profileIcon}
+            alt="Revenues Icon"
```

Comment:
Small hint: change the alt text to something more fitting.
```suggestion
            alt="Profile image"
```